tests/test_classification_accs.py

```diff
@@ -28,3 +27,38 @@ def test_evaluate_multilabel_accuracy(self):
         pred = np.array([0, 1, 1, 1, 1, 1])
         y_pred = {'prediction': pred}
         self.assertAlmostEqual(evaluate_multilabel_accuracy(y_true, y_pred), 0.9090909091, 10)
+
+    def test_evaluate_top_k_accuracy(self):
+        # vacuous case: no error
+        true = np.array([1, 1, 1, 0, 0, 0])
+        y_pred_true = {'prediction': true}
+        self.assertEqual(evaluate_top_k_accuracy(true, y_pred_true, k=1), 1.0)
```

Comment:
It would be good to add another assertion in each case here to check against the actual value that the `sklearn` implementation returns as well

Reply:
Thanks for the feedback! Please see https://github.com/mindsdb/mindsdb_evaluator/pull/16/commits/a9b7358935582ef447da5e82a9224902afff1915 which includes these checks too 😄